lib/p2p/Peer.ts

```diff
@@ -87,9 +87,16 @@ class Peer extends EventEmitter {
     }
   }
 
+  get pubKey(): string {
+    if (this.host) {
+      return this.host.pubKey;
+    } else {
+      return '';
+    }
+  }
+
   constructor() {
     super();
-
```

Comment:
Personally I like having this blank line between `super()` and the rest of the constructor, and it's a style I've seen often elsewhere too, including everywhere else in the code base currently. Let's keep it consistent.